pkg/utils/testing.go

```diff
@@ -85,6 +76,37 @@ func (fs *FakeFilesystem) Use() func() {
 	}
 }
 
+// TODO: Remove writing pci.ids file once ghw is mocked
+// This is to fix the CI failure where ghw lib fails to
+// unzip pci.ids file downloaded from internet.
+func writePciIds(fs *FakeFilesystem) {
+	//nolint: gocritic
+	pciData, err := os.ReadFile("/usr/share/hwdata/pci.ids")
+	if err != nil {
+		if errors.Is(err, os.ErrNotExist) {
+			// use the test data if we run inside a container where we don't have the pci ids file
+			dir, err := os.Getwd()
+			if err != nil {
+				panic(fmt.Errorf("error getting working directory: %s", err.Error()))
+			}
+			//nolint: gocritic
+			pciIdsPath := filepath.Join(dir, "../../pkg/utils/testdata/pci.ids")
+			pciData, err = os.ReadFile(pciIdsPath)
```

Comment:
i think for tests we can always use this file. less variation when running test.